### ISSUE TITLE
Client Insights Dashboard won't import

### ISSUE THREAD

**Issue by paralovosca** [davidnewhall/unifi-poller#99]
Grafana: 6.3.2
![Screen Shot 2019-08-16 at 00 10 00](https://user-images.githubusercontent.com/42482868/63143002-468d5300-bfba-11e9-8b86-e01d7a115bda.png)

I was able to get influxdb, grafana, plugins and your other dashboards to install.  Only Client Insights fails to import.  I have tried via 10418 code from website.  I have downloaded the json to import and I have tried copying the json contents to the import page.  All give me the red bubble with "Failed to import dashboard".
Let me know if you need anything to assist with troubleshooting.

**Comment by davidnewhall** [comment]
Glad you find the correct repo to open this issue. HMU on Discord, I'm in there talking to others about UDM right now. :D

**Comment by davidnewhall** [comment]
Can you check the Grafana log? It may have clues. Usually located at `/var/log/grafana/grafana.log` on unix systems or in `<grafana_install_dir>/data/log` on other platforms & manual installs.

**Comment by davidnewhall** [comment]
I know I mentioned this before too, but make sure you have installed the three plugins. https://github.com/davidnewhall/unifi-poller/wiki/Grafana#plugins

**Comment by paralovosca** [comment]
I 'tail -f -n 50 /usr/local/var/log/grafana/grafana.log' and when I attempted an import of the dashboard in question gave me the error "error="Dashboard name cannot be the same as folder".
I had been following the recommendation to create separate folders for each dashboard.  Since this was the first one I attempted to import I had called the first folder the same name as the dashboard.  Even though on subsequent attempts I created other folders, the fact that I had named the same as the dashboard I was trying to import made it fail.  I guess when I moved on to import the remaining dashboards I had shorten the folder names.  Today I deleted the folder in question within grafana and I was able to successfully import the dashboard.  Thanks for your prompt responses!
